front/app/hooks/useEvents.ts

```diff
@@ -7,7 +7,7 @@ type sort = 'newest' | 'oldest';
 
 interface InputParameters {
   projectIds?: string[];
-  futureOnly?: boolean;
+  notPastOnly?: boolean;
```

Comment:
I wonder if `currentAndFutureOnly` would be a better name because it gets rid of the negation, but I am not sure. :crystal_ball:

Reply:
I think your suggestion is better. Thanks. Will edit.